bedita-app/libs/be_thumb.php

```diff
@@ -91,7 +91,14 @@ class BeThumb {
 	 * All known mime types
 	 * internal use (if needed), read from config/mime.types.php
 	 */
-	private $knownMimeTypes = array();
+    private $knownMimeTypes = array();
+    
+    /**
+     * It says if cache dir for thumbnail has been already checked and eventually created.
```

Comment:
[minor] "eventually" means "in the end", it's nothing like Italian's "eventualmente". 🙃 I'd go with "checked and created", simply.

Reply:
You are right :+1: :smile_cat: